TicketBookingBackend/TicketBookingAPI/TicketBookingAPI/Controllers/TicketBookingController.cs

```diff
@@ -46,10 +48,31 @@ public async Task<ActionResult<BusModel>> GetBus(int busId)
         }
 
         [HttpPost("SaveTicket")]
-        public async Task<ActionResult<bool>> SaveTicket(TicketModel ticketModel) 
+        public async Task<ActionResult<bool>> SaveTicket(TicketModel ticketModel)
         {
             var res = await _ticketService.SaveTicket(ticketModel);
             return Ok(res);
         }
+
+        [HttpPost("SaveTicketForGuest")]
+        public async Task<ActionResult<bool>> SaveTicketForGuest(Dictionary<string, object> data)
```

Comment:
This in not correct approach, also it not very clear what it is storing in dictionary